src/components/editMeasure/populationCriteria/baseConfiguration/BaseConfiguration.tsx

```diff
@@ -56,8 +56,9 @@ const BaseConfiguration = () => {
     initialValues: {
       scoring: measure?.scoring || "",
       baseConfigurationTypes: measure?.baseConfigurationTypes || [],
-      patientBasis: measure?.patientBasis || "true",
+      patientBasis: measure?.patientBasis ? "true" : "false",
```

Comment:
The story mentions default value is true

Reply:
@adongare The backend has a default value of true for the field.

Reply:
@adongare I tested create a new qdm measure, the field is set to true, with an existing qdm measure, it will also defaulted to true.

Reply:
Gotcha, we are just converting a boolean to a string. thank you!